packages/frontend/src/lib/Card.svelte

```diff
@@ -0,0 +1,33 @@
+<script lang="ts">
+export let title: string | undefined = undefined;
+export let description: string | undefined = undefined;
+export let classes: string = '';
+
+export let href: string | undefined = undefined;
+</script>
+
+<a class="no-underline" href={href}>
+  <div class="{classes} rounded-md flex-nowrap overflow-hidden" role="region" aria-label={title ?? 'Card'}>
```

Comment:
We are doing this in a few places, often with different parameter names. In Svelte 5 you can do:
```
let {
  class: className = '',
}: {
  class?: string;
} = $props();
```
to make this cleaner, but since we ever only ever do `classes="font-medium mt-4"`, maybe remove the parameter and put the styling in here?